### PR TITLE
Added anchor links

### PR DESCRIPTION
**Pull request by KalleHoppe**
Added anchor links to the sections in the checklist

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=501) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=501) before we can accept your contribution.<br/><br/>:x: KalleHoppe [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=501)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetDocs?pullRequest=501) it.</sub>

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/creating-an-entity-framework-data-model-for-an-asp-net-mvc-application.md [hunk 1]
```diff
@@ -33,15 +33,15 @@ This tutorial series explains how to build the Contoso University sample applica
...
-> * Create an MVC web app
-> * Set up the site style
-> * Install Entity Framework 6
-> * Create the data model
-> * Create the database context
-> * Initialize DB with test data
-> * Set up EF 6 to use LocalDB
-> * Create controller and views
-> * View the database
+> * [Create an MVC web app](#Create-an-MVC-web-app)
```
Comment (Rick-Anderson) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/creating-an-entity-framework-data-model-for-an-asp-net-mvc-application.md:36]: Tag names in XHTML are case sensitive.  You need to make all the letters lower case.

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
Change all to lower case

**Comment by Rick-Anderson** [comment]
@KalleHoppe  thanks for the thought though. I'm sure you can find docs to improve.

**Pull request closed without merging**